lib/Parser/Parse.cpp

```diff
@@ -2447,6 +2523,146 @@ IdentPtr Parser::ParseImportOrExportFromClause(bool throwIfNotFound)
     return moduleSpecifier;
 }
 
+template<bool buildAST> 
+ParseNodePtr Parser::ParseDefaultExportClause()
+{
+    Assert(m_token.tk == tkDEFAULT);
+
+    m_pscan->Scan();
+    ParseNodePtr pnode = nullptr;
+    ushort flags = fFncNoFlgs;
+    
+    switch (m_token.tk)
+    {
+    case tkCLASS:
+        {
+            if (!m_scriptContext->GetConfig()->IsES6ClassAndExtendsEnabled())
+            {
+                goto LDefault;
```

Comment:
My instinct is that these switches will simply rot over time if they are not being regularly exercised and so that's why I'd prefer to remove them.